src/agent/src/rpc.rs

```diff
@@ -428,6 +448,7 @@ impl agentService {
 
         resp.status = p.exit_code;
 
+        ctr.exit_status.insert(eid, p.exit_code);
```

Comment:
Why we cached this exit code here?  And  I didn't find any code to do the cleanup except destroying the container, in that case, if there are much more exec, than would it cause memory leak?